akka-actor-tests/src/test/scala/akka/actor/ExtensionSpec.scala

```diff
@@ -98,7 +98,8 @@ class ExtensionSpec extends WordSpec with Matchers {
         """
           akka.extensions = ["akka.actor.MissingExtension"]
         """))
-      EventFilter.error("While trying to load extension [akka.actor.MissingExtension], skipping...").intercept()(system)
+      val t = ()
+      EventFilter.error("While trying to load extension [akka.actor.MissingExtension], skipping...").intercept(t)(system)
```

Comment:
Helps us not having to disable for all akka-actor-tests just for this one point.

Reply:
A separate variable is not needed. This works: `intercept(())`